packages/proxy/test/unit/http/util/service-worker-manager.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('lib/http/util/service-worker-manager', () => {
 
     manager.addInitiatorToServiceWorker({
       scriptURL: 'http://localhost:8080/sw.js',
-      initiatorURL: 'http://localhost:8080/index.html',
+      initiatorOrigin: 'http://localhost:8080/index.html',
```

Comment:
isn't this technically a url? Probably doesn't make a difference to the test, but we should probably make it an origin
```suggestion
      initiatorOrigin: 'http://localhost:8080',
```

Reply:
Good call. Will update.

Reply:
Good call: [`7f90d22` (#28748)](https://github.com/cypress-io/cypress/pull/28748/commits/7f90d228db88b80a6626b12432ea79b1a8d2b515)